### DISCUSSION TITLE
Typo in theme Copenhagen, h6 is 85em

### DISCUSSION THREAD

**Discussion by nueh** [datenstrom/community#885] [Report a bug] [answered]
https://github.com/annaesvensson/yellow-copenhagen/blob/1014b455a27200ff6df10457b3b1b5ac29688f19/copenhagen.css#L118C20-L118C20

There seems to be a typo in line 118, `85em` instead of `0.85em`

```
117   .content h6 {
118       font-size: 85em;
119   }
```

**Comment by markseuffert** [comment, marked as answer]
Thank you for the bug report. Should be fixed now, please try it out.
